[2022-02-15 02:29:15,549] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.test.my_new_project.int_orders_pivoted manual__2022-02-15T02:27:57.149490+00:00 [queued]>
[2022-02-15 02:29:15,566] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.test.my_new_project.int_orders_pivoted manual__2022-02-15T02:27:57.149490+00:00 [queued]>
[2022-02-15 02:29:15,567] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 02:29:15,569] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 02:29:15,570] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 02:29:15,591] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): test.my_new_project.int_orders_pivoted> on 2022-02-15 02:27:57.149490+00:00
[2022-02-15 02:29:15,598] {standard_task_runner.py:52} INFO - Started process 921 to run task
[2022-02-15 02:29:15,607] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'test.my_new_project.int_orders_pivoted', 'manual__2022-02-15T02:27:57.149490+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmp60qmqogr', '--error-file', '/tmp/tmpcpvi2g59']
[2022-02-15 02:29:15,610] {standard_task_runner.py:77} INFO - Job 76: Subtask test.my_new_project.int_orders_pivoted
[2022-02-15 02:29:15,771] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.test.my_new_project.int_orders_pivoted manual__2022-02-15T02:27:57.149490+00:00 [running]> on host de7a240d0d46
[2022-02-15 02:29:15,857] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=test.my_new_project.int_orders_pivoted
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T02:27:57.149490+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T02:27:57.149490+00:00
[2022-02-15 02:29:15,859] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 02:29:15,862] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json test --target dev --models int_orders_pivoted             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 02:29:15,882] {subprocess.py:85} INFO - Output:
[2022-02-15 02:29:23,030] {subprocess.py:89} INFO - 02:29:23  Running with dbt=1.0.1
[2022-02-15 02:29:23,437] {subprocess.py:89} INFO - 02:29:23  [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2022-02-15 02:29:23,438] {subprocess.py:89} INFO - There are 2 unused configuration paths:
[2022-02-15 02:29:23,439] {subprocess.py:89} INFO - - models.jaffle_shop.staging
[2022-02-15 02:29:23,440] {subprocess.py:89} INFO - - models.jaffle_shop.marts.core
[2022-02-15 02:29:23,440] {subprocess.py:89} INFO - 
[2022-02-15 02:29:23,473] {subprocess.py:89} INFO - 02:29:23  Found 7 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 02:29:23,484] {subprocess.py:89} INFO - 02:29:23
[2022-02-15 02:29:23,487] {subprocess.py:89} INFO - 02:29:23  [WARNING]: Nothing to do. Try checking your model configs and model specification args
[2022-02-15 02:29:24,069] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 02:29:24,122] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=test.my_new_project.int_orders_pivoted, execution_date=20220215T022757, start_date=20220215T022915, end_date=20220215T022924
[2022-02-15 02:29:24,192] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 02:29:24,262] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
